config/style.yml

```diff
@@ -147,41 +141,17 @@ Style/GuardClause:
   Description: 'Check for conditionals that can be replaced with guard clauses'
   Enabled: false
 
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
-Style/MethodDefParentheses:
-  Description: >-
-    Checks if the method definitions have or don't have
-    parentheses.
-  Enabled: false
-
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
-Style/NegatedIf:
-  Description: >-
-    Favor unless over if for negative conditions
-    (or control flow or).
-  Enabled: false
```

Comment:
Wow didn't realise this was disabled, I assume this also comes back as enabled.

Reply:
Yes, that's right - it's implied in the commit message.